ui-core/src/styles/inputs/timePicker.css

```diff
@@ -1,9 +1,11 @@
-.time-picker {
-  .time-input {
-    cursor: pointer;
+@-moz-document url-prefix() {
```

Comment:
What is this `@-moz-document url-prefix` ? Is it a class specific for mozilla ?
Anyway, can you move this code in .time-picker l7 ?

Reply:
yes, the selector only applies to firefox.
I tried to move this in the .time-picker block but it just breaks the style.

I think it doesn’t work if it’s not at the top level